src/System.Management.Automation/engine/DscResourceInfo.cs

```diff
@@ -1,6 +1,6 @@
 //-----------------------------------------------------------------------
 // <copyright company="Microsoft">
-//    Copyright (C) 2013 Microsoft Corporation
+//    Copyright (c) Microsoft Corporation
```

Comment:
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."